src/goTest.ts

```diff
@@ -71,6 +73,32 @@ export function testAtCursor(timeout: string) {
 }
 
 /**
+* Executes the unit test or main at the primary cursor using `go run/test`. Output
+* is sent to the 'Go' channel.
+* 
+* @param timeout a ParseDuration formatted timeout string for the tests.
+*
+* TODO: go test returns filenames with no path information for failures,
+* so output doesn't produce navigable line references.
+*/
+export function runAtCursor(timeout: string) {
```

Comment:
IMO it would be clearer if each function had a single conceptual responsibility. A function to run a program/main, and a function to run a test. I'm not quite seeing the benefit of grouping these together.